playground/android/build.gradle

```diff
@@ -18,26 +18,18 @@ buildscript {
 
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath 'com.android.tools.build:gradle:7.1.1'
+        classpath("com.facebook.react:react-native-gradle-plugin")
+        classpath 'com.android.tools.build:gradle:7.3.1'
         classpath("de.undercouch:gradle-download-task:5.0.1")
```

Comment:
we probably no longer need this `classpath("de.undercouch:gradle-download-task:5.0.1")`